content/en/the-launch/sprints/2/day-plan/index.md

```diff
@@ -6,4 +6,16 @@ menu_level = ['sprint']
 weight = 3
 backlog= 'Module-The-Launch'
 backlog_filter= 'Week 2'
+[[blocks]]
+name="Demo Time"
+src="https://cyf-pd.netlify.app/blocks/demo-time/readme/"
+[[blocks]]
+name="Retrospective"
+src="https://cyf-pd.netlify.app/blocks/retrospective/readme/"
+[[blocks]]
+name="Sprint Planning"
+src="https://cyf-pd.netlify.app/blocks/sprint-planning/readme/"
+[[blocks]]
+name="Project work"
+src="https://cyf-curriculum.netlify.app/portfolio/sprints/2/day-plan/#project-work"
```

Comment:
```suggestion
src="portfolio/blocks/project"
```

You can call any block in our curriculum like this.

Reply:
And you can offer revisions through a PR! 😄 🙏 

They will go through our open review process.  I definitely encourage you to offer revisions, or additional versions, of these blocks in particular:

### Portfolio

https://github.com/CodeYourFuture/curriculum/blob/main/content/en/portfolio/blocks/stand-up/index.md

https://github.com/CodeYourFuture/curriculum/blob/main/content/en/portfolio/blocks/break-it-down/index.md

https://github.com/CodeYourFuture/curriculum/blob/main/content/en/portfolio/blocks/next-steps/index.md

## General
https://github.com/CodeYourFuture/curriculum/tree/main/content/en/blocks/energiser
https://github.com/CodeYourFuture/curriculum/tree/main/content/en/blocks/retro